pandas/tests/indexing/test_indexing.py

```diff
@@ -631,6 +631,15 @@ def test_index_type_coercion(self, indexer):
             indexer(s2)["0"] = 0
             assert is_object_dtype(s2.index)
 
+    def test_setitem_one_element_list(self):
+        # GH#52825
+        x_values = ["x_value"]
+        y_values = ["y_values"]
+        df = DataFrame(columns=["x", "y"])
+        df.loc[:, "x"] = x_values
+        df.loc[:, "y"] = y_values
+        assert df.size == 2
```

Comment:
can you explicitly construct expected and check for a match

Reply:
Done